support/hibernate-javax/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+@file:Suppress("VulnerableLibrariesLocal")
+
+dependencies {
+    compileOnly(libs.spring.boot2.starter)
+    compileOnly(libs.javax.hibernate.core)
+    compileOnly(projects.jpqlDsl)
+    compileOnly(projects.jpqlQueryModel)
+    compileOnly(projects.jpqlRender)
+
+    testImplementation(libs.spring.boot2.starter)
+    testImplementation(libs.javax.hibernate.core)
+    testImplementation(projects.jpqlDsl)
+    testImplementation(projects.jpqlQueryModel)
+    testImplementation(projects.jpqlRender)
+}
+
```

Comment:
```suggestion
kover {
     excludeInstrumentation {
         packages("org.hibernate.*")
     }
}
```
Adding this line seems to resolve the test report failure.
When kover performs a bytecode instrument to measure coverage, it conflicts with the bytecode instrument performed by hibernate.

Reply:
I have applied the review.